neuropredict-frontend/src/index.css

```
@@ -2,772 +2,254 @@
+   cool-grey family. Radii follow one rule: pill for interactive, 16px for
+   surfaces, 8px for form fields.
+   --------------------------------------------------------------------------- */
+:root {
+  --accent: #667eea;
+  --accent-deep: #764ba2;
+  --ink-900: #0b0d12;
+  --ink-800: #12151d;
+  --ink-700: #1a1e29;
+  --border: rgba(255, 255, 255, 0.09);
+  --ease: cubic-bezier(0.16, 1, 0.3, 1);
+  color-scheme: dark;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)